src/audio/google/Kconfig

```diff
@@ -24,14 +24,6 @@ config COMP_GOOGLE_RTC_AUDIO_PROCESSING
 	  This component takes raw microphones input and playback reference
 	  and outputs an echo-free microphone signal.
 
```

Comment:
>> The internals of the AEC library are floating point already.
just a comment: from my measurements of the AEC binary I got - 16bit API used to be faster than 32bit.

Reply:
Is that maybe polluted by conversion overhead?  Or maybe it's a cache footprint issue?  My understanding is that the behavior is identical once you get across the API boundary.  @lkoenig might have input.

Reply:
Internally the int16 are converted back to float32. If the int16 is faster than the float32, we might need to look into it.

Reply:
@singalsu any comments here ?

Reply:
From AEC quality point of view I think 32 bit float is a good choice over 16 bit integer. The raw Mic signal level is typically low since there is headroom for very loud ambient noise.